NXCON-259: Add How To page for Studio configuration (#2694)
nuxeo/doc.nuxeo.com-platform-spaces: src/studio/nuxeo-studio/tutorials/how-to-configure-studio-for-using-knowledge-enrichement.md

* **ID:** `imageDescriptionAndNamedEntitiesBtn`
* **Label:** Describe Image
* **Category:** `DOCUMENT_ACTIONS`
* **Type:** button
* **Operation:** `ImageDescription`

**Text Classification Button**
* **ID:** `TextClassification`
* **Label:** Classify Text
* **Category:** `DOCUMENT_ACTIONS`
* **Type:** button
* **Operation:** `TextClassification`

### 3.2 Button Configuration Details

When configuring the buttons, ensure the following parameters are set:

1. **Slot:** `Document Actions`
2. **Element:** Create an element named `nuxeo-operation-button-with-spinner`.
3. **Icons:** Add the desired icon representing how the button will look in the UI.
4. **Operation:** Bind the appropriate automation script (from Step 2) you want to call through the button.
5. **Labels and Notifications:** Define the text displayed when a user hovers over the button to understand its utility.
6. **Activation Filter (Optional but recommended):** Set visibility conditions so the button only appears for appropriate file types.
   * *Example for Images:* `document.type == 'Picture'`
   * *Example for Text:* `document.type == 'File'` (or custom Note type)

## Step 4: Test the Integration

> Once all configurations are complete in Studio, you must deploy and test them on your server.

Reviewer: ```suggestion
Once all configurations are complete in Studio, perform the following steps to deploy and test them on your server. 
```

Author: fixed

1. **Save and Push:** Save your changes in Studio and push the project.

2. **Deploy to Server:** Run the following command on your server to install the updated jar files with your pushed changes:

````bash
   nuxeoctl mp-install <studio-project-name>
````

```bash
   nuxeoctl start
```

4. **Hot Reload:** Use the browser extension Dev Tools to perform a Hot Reload of your UI.

![Use this Hot reload button from Dev tools to sync latest changes in Studio project](.nx_assets/002a1841-fecb-467e-ae34-82de50b78551.png)

> 5. **Execute Action:** Navigate to an uploaded document (PDF/Docx for text, JPG/JPEG for images) and click the respective custom button (e.g., `Classify Text` or `Describe Image`).

Reviewer: ```suggestion
5. **Execute Action:** Navigate to an uploaded document (PDF/Docx for text, JPG/JPEG for images) and click the respective custom button (for example, `Classify Text` or `Describe Image`).
```

Author: fixed
